Things_You_Should_Know/things_you_should_know.md

```diff
@@ -0,0 +1,400 @@
+
+To view the Docker log using tail, access the console, switch into the LogFiles directory, and run this command:
+
+`tail 2017_09_05_RD*0FA_docker.log`
+
+In this example, I'm viewing the Docker log for September 5. Note that I'm also using a wildcard replacement for the machine ID (the RD number) so that I don't have to type all the characters.
+
+> **Tip:** 
+> 
+> You can pipe the output of tail to a new file in case you want to download it. Simply append "> filename.txt" to the command above.
+
+### You can discover and download the latest Docker logs using Kudu
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+**App Service** has an API that allows you to easily see the current Docker log details (such as the filename, etc.) and also download the current Docker logs in Zip format.
+
+To see details on the current Docker logs in JSON format, you can use this URL:
+
+`https://<yout-app-name>.scm.azurewebsites.net/api/logs/docker`
+
+You can get to this easily by going to [**Kudu** (Advanced Tools)](https://github.com/projectkudu/kudu/wiki) and then appending "/api/logs/docker" to the URL. The output of this will be a JSON response with the most relevant and current Docker logs.
+
+If you want to download the logs shown in the above API in Zip format, append "zip" to the URL. For example:
+
+`https://<yout-app-name>.scm.azurewebsites.net/api/logs/docker/zip`
+
+
+### You can view the most recent entries in your Docker log in the Azure portal
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+To make it easier to see what's going on with your Docker container, we show you the latest entries in the Docker log in the Azure portal. Simply click on **Container Settings** in the portal menu and you'll see the Docker log.
+
+You can click the Copy button to copy the Docker log. You can also download the full Docker log by clicking on Download Logs.
+
+### If your container takes a long time to start, increase the start time limit
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+**App Service** will wait 230 seconds for a container to start, initialize and respond to an HTTP request. **App Service** interprets this first response to an HTTP request as an idication that your app has initialized and is ready to recive further requests. If the timout of 230 seconds is reached with no response from the app, then it's considerd a failure to start. 
+
+Some containers might need more than 230 seconds to start, the default timeout can be overridden through the `WEBSITES_CONTAINER_START_TIME_LIMIT` app setting. This app setting can be set to a maximum of 1800 seconds.
```

Comment:
It would be nice to mention that customers should really investigate why apps are taking longer than 230 seconds to start....that could be the sign of something worse.  Customers should review memory and CPU consumption, and avoid doing large DB queries at initialization.